SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -123,7 +123,8 @@ void SSpatialGDKSimulatedPlayerDeployment::Construct(const FArguments& InArgs)
 									SNew(SEditableTextBox)
 									.Text(FText::FromString(ProjectName))
 									.ToolTipText(FText::FromString(FString(TEXT("The name of the SpatialOS project."))))
-									.IsEnabled(false)
+									.OnTextCommitted(this, &SSpatialGDKSimulatedPlayerDeployment::OnProjectNameCommitted)
```

Comment:
Should really rename this `SSpatialGDKSimulatedPlayerDeployment` at some point

Reply:
Yeah I think we should, but I'd suggest we do that after the feature branch is more or less settled otherwise pulling out functionality becomes a nightmare.